code/game/machinery/telecomms/computers/message.dm

```diff
@@ -470,8 +470,12 @@
 		return INITIALIZE_HINT_LATELOAD
 
 /obj/item/paper/monitorkey/proc/print(obj/machinery/telecomms/message_server/server)
-	info = "<center><h2>Daily Key Reset</h2></center><br>The new message monitor key is '[server.decryptkey]'.<br>Please keep this a secret and away from the clown.<br>If necessary, change the password to a more secure one."
-	info_links = info
+	info = {"
+              Daily Key Reset
+The new message monitor key is 'server.decryptkey'.

```

Comment:
You'll want to keep the squarebrackets around `server.decryptkey`, since this is a variable insert.

Reply:
ah thanks  missed that one.  After doing about 20 of those files at 2 am my mind gets warped